tests/MMS/wave-1d/wave.cxx

```diff
@@ -62,7 +62,11 @@ class Wave1D : public PhysicsModel {
     
     /*this assumes equidistant grid*/
     int nguard = mesh->xstart;
-    coord->dx = Lx/(mesh->GlobalNx - 2*nguard);
+
+    Field2D dx=Lx/(mesh->GlobalNx - 2*nguard);
+    coord->dx = dx;
+    dx.setLocation(CELL_XLOW);
+    coord->dx = dx;
```

Comment:
Why the assignment both before and after `dx.setLocation()`?

Reply:
I added some comments in the file.
This way several locations can be set.